spec/reporting_helpers.rb

```diff
@@ -40,4 +40,35 @@ def june_2021
       one_month_ago: now - 1.month
     }
   end
+
+  def current_month
+    timezone = Time.find_zone(Period::REPORTING_TIME_ZONE)
+    year = Date.today.year
+    month = Date.today.month
+    month_date = timezone.local(year, month, 1, 0, 0, 0) # Beginning of the month
+    {
```

Comment:
Can move this to a method which accepts now/month_date and returns this map

Reply:
Can default to june_2021

Reply:
hmm, yeah 👍

Reply:
The `month_date` function is not working in the semaphore test setup. I am looking into it now

Reply:
The `"June 30 2021 23:00 IST"` was set for a reason Kini. Check out the comment above that.